common/SigUtil.cpp

```diff
@@ -232,6 +232,7 @@ void requestShutdown()
         signalLog(buf + i + 1);
     }
 
+#endif // !MOBILEAPP
```

Comment:
But that's what moving this function tried to avoid. The function is blocked in `#if !MOBILEAPP`, and we need to call on mobile, so we either move it out, or we exclude it, like I do here.